### ISSUE TITLE
Could not find AMGX_LIBRARIES using the following names: amgxsh

### ISSUE THREAD

**Issue by xdquan** [barbagroup/AmgXWrapper#41]
Is there anyone meet the same problem like me?

**Comment by piyueh** [comment]
Could you provide the command you used to configure CMake and the outputs?

**Comment by xdquan** [comment]
> Could you provide the command you used to configure CMake and the outputs?

yeah,of course. Thanks a lot. Your reply means a lot for me. the information I used are as follows：
cmake \
    -D CMAKE_INSTALL_PREFIX=/home/super2/AmgXWrapper \
    -D PETSC_DIR=/home/super2/下载/petsc \
    -D PETSC_ARCH=test \
    -D CUDA_DIR=/usr/local/cuda-11.8 \
    -D AMGX_DIR=/home/super2/amgx \
/home/super2/下载/AmgXWrapper-master
the results are as follows:
CMake Error at CMakeLists.txt:80 (find_library):
  Could not find AMGX_LIBRARIES using the following names: amgxsh


-- Configuring incomplete, errors occurred!

And I used the  cuda-11.8 ,the amgx was cloned using the --recursive option and can run the example successfully.

**Comment by piyueh** [comment]
Were you able to find `libamgxsh.so` under the folder `/home/super2/amgx`? The error message means CMake was not able to find `libamgxsh.so`. (If you would like, you can also dump all files and folders under `/home/super2/amgx` to a file and upload the file here. For example, do `tree /home/super2/amgx > log.txt 2>&1` and then upload `log.txt` here.)
